code/drasil-gool/lib/GOOL/Drasil/LanguageRenderer/SwiftRenderer.hs

```diff
@@ -1161,16 +1161,17 @@ swiftModDoc desc as date m = m : [desc | not (null desc)] ++
   [swiftDocCommandInit ++ swiftAuthorDoc ++ swiftDocCommandSep ++ stringList as 
     | not (null as)]
   ++ [swiftDocCommandInit ++ swiftDateDoc ++ swiftDocCommandSep ++ date 
-    | not (null date)]
+    | not (null date)] ++ [swiftDocCommandInit ++ swiftDrasilVersion ++ version drasilMeta]
 
 swiftDocCommandInit, swiftDocCommandSep, swiftParamDoc, swiftRetDoc,
-  swiftAuthorDoc, swiftDateDoc :: String
+  swiftAuthorDoc, swiftDateDoc, swiftDrasilVersion :: String
 swiftDocCommandInit = "- "
 swiftDocCommandSep = ": "
 swiftParamDoc = "Parameter"
 swiftRetDoc = "Returns"
 swiftAuthorDoc = "Authors"
 swiftDateDoc = "Date"
+swiftDrasilVersion = "Generated by Drasil "
```

Comment:
I think this string is duplicated, but we can share it.  Also, I think it would be good to call this a 'watermark' somehow for easier searching later.

Reply:
I will put that in utils and name it `watermark`.